plugins/modules/vmware_vc_infraprofile_info.py

```diff
@@ -0,0 +1,235 @@
+requirements:
+- python >= 2.6
+- PyVmomi
+- vSphere Automation SDK
+options:
+    decryption_key:
+      description:
+      - decryption_key argument for while doing import profile task as of now its not taken into account form API team.
+      type: str
+      required: False
+    encryption_key:
+      description:
+      - encryption_key argument for while doing import profile task as of now its not taken into account form API team.
+      type: str
+      required: False
+    api:
+      description:
+      - which API need to be executed(list,export,import,validate)
+      type: str
+      required: False
+      choices: [ list, export, import, validate ]
+    config_path:
+      description:
+      - Config file path which contains infra profile config JSON data, supports both relative and absolute path.
+      - This parameter is required only when C(import),C(validate) APIs are being used.
+      type: str
+      required: False
+    profiles:
+      description:
+      - A list of profile names to be exported, imported, and validated.
+      - This parameter is not required while running for List API, not for C(export),C(import) and C(validate).
+      type: str
+      required: False
+    description:
+      description:
+      - Description of about encryption or decryption key
+      type: str
+      required: False
+extends_documentation_fragment:
+- community.vmware.vmware_rest_client.documentation
+'''
+
+EXAMPLES = r'''
+- name: Get information about VC infraprofile
+  vmware_vc_infraprofile_info:
+    hostname: '{{ vcenter_hostname }}'
+    username: '{{ vcenter_username }}'
+    password: '{{ vcenter_password }}'
+    validate_certs: False
+  delegate_to: localhost
+
+- name: export Vcenter appliance infra profile config
+  vmware_vc_infraprofile_info:
+    hostname: '{{ vcenter_hostname }}'
+    username: '{{ vcenter_username }}'
+    password: '{{ vcenter_password }}'
+    validate_certs: False
+    api: "export"
+    profiles: "ApplianceManagement"
+  delegate_to: localhost
+
+- name: validate Vcenter appliance infra profile config
+  vmware_vc_infraprofile_info:
+    hostname: '{{ vcenter_hostname }}'
+    username: '{{ vcenter_username }}'
+    password: '{{ vcenter_password }}'
+    validate_certs: False
+    validate_certs: False
+    api: "validate"
+    profiles: "ApplianceManagement"
+    config_path: "export.json"
+
+- name: import Vcenter appliance infra profile config
+  vmware_vc_infraprofile_info:
+    hostname: '{{ vcenter_hostname }}'
+    username: '{{ vcenter_username }}'
+    password: '{{ vcenter_password }}'
+    validate_certs: False
+    validate_certs: False
+    api: "export"
+    profiles: "ApplianceManagement"
+    config_path: "export.json"
+  delegate_to: localhost
+  '''
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.community.vmware.plugins.module_utils.vmware_rest_client import VmwareRestClient
+import json
+import os
+import time
+
```

Comment:
Please add `RETURN` block. It tells user what kind of data the module is going to return in advance (without running the module).